mapper/tedge_mapper/src/collectd_mapper/error.rs

```diff
@@ -3,19 +3,17 @@ use tokio::sync::mpsc::error::SendError;
 #[derive(thiserror::Error, Debug)]
 pub enum DeviceMonitorError {
     #[error(transparent)]
-    MqttClientError(#[from] mqtt_client::MqttClientError),
+    MqttClient(#[from] mqtt_client::MqttClientError),
```

Comment:
That's a really weird name for an error. The "failure" aspect is missing from the name. I personally prefer the earlier names with `Error` suffixes. Probably not for the errors that already has another negative term in them like "Invalid".

Reply:
I checked https://doc.rust-lang.org/src/std/io/error.rs.html, all error names are negative (e.g. TooManyLinks).
I think we had a discussion while implementing sm-mapper, and we decided to prefix `From`. See https://github.com/thin-edge/thin-edge.io/blob/main/mapper/tedge_mapper/src/sm_c8y_mapper/error.rs#L54. Here the name is `FromMqttClient`.
I prefer `FromMqttClient`.